services/device-registry-base/src/main/java/org/eclipse/hono/service/management/device/Device.java

```diff
@@ -54,6 +54,9 @@
     @JsonFormat(with = JsonFormat.Feature.ACCEPT_SINGLE_VALUE_AS_ARRAY)
     private List<String> memberOf = new LinkedList<>();
 
+    @JsonInclude(value = Include.NON_EMPTY)
+    private String mapper = "";
```

Comment:
does this really need to be configured at the device instead of the tenant level?